apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -106,6 +109,20 @@ export class BulkScriptAction extends BaseScriptAction {
       }
     }
 
+    const traceFilePath: string = this._traceParameter.value
+      ? path.join(this.rushConfiguration.rushJsonFolder, `${this.actionName}.${Date.now()}.trace.json`)
```

Comment:
Should this go in the `common/temp` folder? Or perhaps the `--trace` parameter should take a path to the output file. That'd probably be more useful on CI machines.